rotkehlchen/chain/evm/base_contracts.py

```diff
@@ -0,0 +1,11 @@
+import json
+from typing import Final
+
+from rotkehlchen.chain.evm.contracts import EvmContract
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+BALANCE_SCANNER: Final = EvmContract(  # defined here only until we merge bugfixes to develop  # noqa: E501
```

Comment:
can you make a note to do this and not forget?

Reply:
Done here https://github.com/orgs/rotki/projects/11?pane=issue&itemId=88901029